src/lib/components/Callback/utils.js

```diff
@@ -0,0 +1,29 @@
+import { getURLSearchParams } from '../../utils';
+import superset from '@onaio/superset-connector';
+
+/**
+ * Recursive method that trys to login user to superset
+ * If authentication is successeful redirect to dashboard
+ * Else If Number of retrys are less than 10 attempt again
+ * Else Inform user and redirect to home
+*/
+export const supersetLogin = (config, history, n=10) => {
+    superset.authZ(config, result => {
+    if (result.status === 200) {
+      console.log('User logged into superset', config.base);
```

Comment:
I always cringe when production apps output to the console.  Do we care about that here?

Reply:
@moshthepitt sure, logs are anti-patterns. They help establish issues that arise when superset isn't loading as expected. What would you advise?

Reply:
Yeah they are useful for that.  Ideally if we want to be well-behaved we should not expose this to users (even in the console).  So we would need another way to log these.  [Something like sentry](https://docs.sentry.io/platforms/javascript/?platform=browsernpm#captureconsole) maybe?